service/app/controllers/usermanagement/UserController.java

```diff
@@ -115,11 +116,15 @@
   }
 
   public Promise<Result> searchUser() {
+    final String requestedFields = request().getQueryString(JsonKey.FIELDS).toLowerCase();
```

Comment:
@amolnin @amit-tarento What is fields is not passed then will it give null pointer exception?